### PR TITLE
Added lazy loading to the MALDI ion table

### PR DESCRIPTION
**Pull request by Mailaender**
This is meant to improve performance.

### CODE REVIEW

**Comment by eselmeister** [comment]
When opening a file in the MALDI-TOF perspective, the m/z list stays empty. Is that intended?

Bildschirmfoto von 2022-02-19 11-03-36

**Comment by Mailaender** [comment]
Updated.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.swt.ui/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -28,7 +28,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.nebula.visualization.widgets;bundle-version="1.0.0",
  org.eclipse.e4.core.services;bundle-version="2.0.0",
  org.eclipse.swtchart;bundle-version="0.8.0",
- org.eclipse.swtchart.extensions;bundle-version="0.8.0"
+ org.eclipse.swtchart.extensions;bundle-version="0.8.0",
+ org.eclipse.chemclipse.ux.extension.ui
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.swt.ui/META-INF/MANIFEST.MF]: Please define the bundle-version.

**Comment by eselmeister** [comment]
The lazy modus works now. But the table is not updated correctly, if another mass spectrum was loaded previously.

Bildschirmfoto vom 2022-02-28 09-33-53

How to test this:

1. Load e.g. a chromatogram via the data analysis perspective.
2. Then navigate to the MALDI-TOF perspective.
3. Select a scan in the previously loaded chromatogram, it shall be displayed in the m/z list.
4. Then load the MALDI-TOF file. The previously selected scan remains in the list.

**Comment by Mailaender** [comment]
I removed the broken lazy loading.

**Pull request merged**